build.gradle

```diff
@@ -17,6 +17,7 @@ repositories {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    implementation "org.jetbrains.kotlin:kotlin-reflect:1.4.0"
```

Comment:
Can you elaborate on what this dependency adds to the project?

Reply:
This is kinda roflyanochka.

I've upgraded the Kotlin version on the line above. Then I launched compiling via Gradle and it automatically tried to use this library with previous Kotlin version, that is obviously an error, because versions must match, and there was an appropriate message for that "consider writing this dependency version explicitly: ...". And I've just done it as it had said.